src/features/cseMachine/animationComponents/base/AnimationUtils.ts

```diff
@@ -0,0 +1,130 @@
+import { Binding } from '../../components/Binding';
+import { Frame } from '../../components/Frame';
+import { Visible } from '../../components/Visible';
+import { AnimationConfig } from './Animatable';
+
+/** Omits the index signature `[key: string]: any;` from type `T` */
+export type RemoveIndex<T> = {
+  [K in keyof T as string extends K
+    ? never
+    : number extends K
+    ? never
+    : symbol extends K
+    ? never
+    : K]: T[K];
+};
```

Comment:
Should we move this to `src/commons/utils/TypeHelpers.ts`?

Reply:
Didn't know about this file, updated in the latest commit